cat.md

```diff
@@ -67,6 +68,21 @@ Cat Semantics
          <cat-vow> VOWADDR </cat-vow>
       requires (INK *Rat SPOT) <Rat (ART *Rat RATE)
 
+    syntax CatStep ::= "emitBite" String Address
+    syntax Event ::= Bite(ilk: String, urn: Address, ink: Wad, art: Wad, tab: Wad, flip: Address, id: Int)
+ // ------------------------------------------------------------------------------------------------------
+    rule <k> ID:Int ~> emitBite ILK URN => ID ... </k>
+         <frame-events> _ => ListItem(Bite(ILK, URN, minRat(INK, LUMP), minRat(ART, minRat(INK, LUMP) *Rat ART /Rat INK), minRat(ART, minRat(INK, LUMP) *Rat ART /Rat INK) *Rat RATE *Rat CHOP, address(Flip ILK), ID)) </frame-events>
+         <cat-ilks>...
+           ILK |-> Ilk(... chop: CHOP, lump: LUMP)
+         ...</cat-ilks>
+         <vat-urns>...
+           { ILK, URN } |-> Urn( INK, ART )
+         ...</vat-urns>
+         <vat-ilks>...
+           ILK |-> Ilk(... rate: RATE)
+         ...</vat-ilks>
```

Comment:
I don't like how we're recalculating the values used in the inital `bite` semantics here in the event. Any way we can re-use them? Maybe using `#fun` cleverly?